core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -302,6 +302,53 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
+  def setProjectSpendConfiguration(projectName: RawlsBillingProjectName, datasetName: String): Future[Int] = {
+
+    validateBigQueryDatasetName(datasetName)
+
+    requireProjectAction(projectName, SamBillingProjectActions.alterSpendReportConfiguration) {
+      val bqService = bqServiceFactory.getServiceFromCredentialPath(pathToCredentialJson, GoogleProject(projectName.value))
```

Comment:
in what cases can that happen?

Reply:
v2 billing projects don't have their own google project. This will be the default for new projects starting with PPW. Workspaces' google projects will have auto-generated IDs.